src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Commands/Checkout/ChooseShippingMethod.xml

```diff
@@ -17,7 +17,7 @@
 >
     <class name="Sylius\Bundle\ApiBundle\Command\Checkout\ChooseShippingMethod">
         <attribute name="shippingMethodCode" serialized-name="shippingMethod">
-            <group>shop:cart:select_shipping_method</group>
+            <group>shop:order:select_shipping_method</group>
```

Comment:
following the example of order item, it seems to me, that
```suggestion
            <group>shop:shipment:update</group>
```

 would be a better group for this endpoint